### ISSUE TITLE
Unable to stop download

### ISSUE THREAD

**Issue by marcoil** [xpipe-io/xpipe#396]
In the browser, connected to an SSH host, I started a large file download by error and I haven't been able to find a way to stop it. I can see the download the download process in the bottom but it's not interactive, and trying to close the connection tag doesn't seem to do anything.

I had to kill the background daemon to stop the download.

**Comment by crschnick** [comment]
Yeah that is a limitation of the shell-based approach right now. I plan to improve that situation in the future.

**Comment by crschnick** [comment]
This is now implemented in the v14 release
